### ISSUE TITLE
pickadate.js does not support strict mode

### ISSUE THREAD

**Issue by aleksanderzubkov** [amsul/pickadate.js#764]
pickadate.js has problems with minificators because of undefined variable namesCount assignment:

https://github.com/amsul/pickadate.js/blob/master/lib/picker.js#L540
```
for ( i = 0, namesCount = names.length; i < namesCount; i += 1 ) {
```
should be
```
for ( i = 0; i < names.length; i++ ) {
```

**Comment by amsul** [comment]
@DanielRuf seems like a simple fix :)

**Comment by DanielRuf** [comment]
Theoretically `i` and `namesCount` should both use `var`.

**Comment by amsul** [comment]
`i` is already there though. So `namesCount` is the only problem as it is

**Comment by DanielRuf** [comment]
Sure it is normally better to move them into the head of the loop (imo).

**Comment by DanielRuf** [comment]
Sure but it is normally better to move them into the head of the loop (imo).

**Comment by amsul** [comment]
@DanielRuf true..it's just super old code. Feel free to fix it as you see fit.
